applicant/templates/static/CSS/styles_app.css

```diff
@@ -18,6 +18,7 @@ body {
 	overflow: hidden;
 	height: -moz-available;
 	height: -webkit-fill-available;
+	height: 94%;
```

Comment:
it still here :(

Reply:
0.o it is here but not above ég er confusion

Reply:
Sjáðu nýjasta commitið